com.unity.render-pipelines.core/Editor/MaterialUpgrader.cs

```diff
@@ -432,9 +433,11 @@ public static void UpgradeProjectFolder(List<MaterialUpgrader> upgraders, HashSe
         /// <param name="flags">Material Upgrader flags.</param>
         public static void Upgrade(Material material, MaterialUpgrader upgrader, UpgradeFlags flags)
         {
-            var upgraders = new List<MaterialUpgrader>();
-            upgraders.Add(upgrader);
-            Upgrade(material, upgraders, flags);
+            using (ListPool<MaterialUpgrader>.Get(out List<MaterialUpgrader> upgraders))
```

Comment:
What's this about?

Reply:
This is an list pool, that we are not generating garbage with the new List<MaterialUpgrader>, as we are calling each material and creating a new List, I saw it while updating the docs.

Reply:
Neat, I should look into that. Didn't know this type of thing existed.